scylla/src/transport/retry_policy.rs

```diff
@@ -135,6 +135,10 @@ impl Default for DefaultRetrySession {
 
 impl RetrySession for DefaultRetrySession {
     fn decide_should_retry(&mut self, query_info: QueryInfo) -> RetryDecision {
+        let cl = match query_info.consistency {
+            LegacyConsistency::Serial(_) => return RetryDecision::DontRetry,
+            LegacyConsistency::Regular(cl) => cl,
+        };
```

Comment:
Maybe the consistency set on retry decision could be `LegacyConsistency`?
Eventually I think we would like to support retrying LWTs as well.

Reply:
It look like java driver handles serial consistencies, maybe we could base our code on it.

Reply:
I think this code is a good enough first step, we can consider LWT in a follow-up